src/tests/unittests/configuration_helper/adapters/test_hdawg8_instrument_adapter.py

```diff
@@ -193,6 +193,14 @@ def test_stop(self):
             dawg.assert_has_calls([call().stop_awg(1)])
             hdawg_adapter.instrument.close()
 
+    def test_close_instrument(self):
+        dawg_instace = MagicMock()
```

Comment:
```suggestion
        dawg_instance = MagicMock()
```
And two more in this method